resources/build-npm.js

```diff
@@ -13,6 +13,18 @@ const {
   showDirStats,
 } = require('./utils.js');
 
+const entryPoints = fs
+  .readdirSync('./src', { recursive: true })
+  .filter((f) => f.endsWith('index.ts'))
+  .map((f) => f.replace(/^src/, ''))
+  .reverse()
+  .concat([
+    'execution/execute.ts',
+    'jsutils/instanceOf.ts',
+    'language/parser.ts',
+    'language/ast.ts',
```

Comment:
These have been chosen by gut feeling out of the exports mentioned in #4074

We need to generally decide on a way forward here:  
* expose those `.js` file?
* instead, export these exports from the parent "nested" entrypoint?
* export them from the main entrypoint?

Reply:
I am not sure we should export any files individually.

Reply:
Labelling a function with `export` but burying it so users would have to deep import it was a way to allow entryway into our private API without issuing any semver guarantee => a convenience to power users, so to speak.

With explicit exports, with option 1 and 2, we kind of lose that distinction, and it's much more difficult to garden off these bits as a private API that happens to be exposed.

So I vote that we examine each in term, and make the somewhat difficult decision of whether to support semver on them, and in that case go with option C, export from main entrypoint, or no longer expose them at all.